gems/pending/MiqVm/test/localVm.rb

```diff
@@ -1,11 +1,7 @@
-
-$LOAD_PATH.push("#{File.dirname(__FILE__)}/..")
-$LOAD_PATH.push("#{File.dirname(__FILE__)}/../..")
-
-require_relative '../../bundler_setup'
+require 'bundler_setup'
```

Comment:
I think the bundler_setup stuff has to stay relative for now.  That is the guy that sets up gems/pending on the load_path (and other bundler stuff)